packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2904,6 +2923,15 @@ class InputDecoration {
   /// If [InputDecorationTheme.hintFadeDuration] is null defaults to 20ms.
   final Duration? hintFadeDuration;
 
+  /// Whether the input field's height should always be greater than or equal to
+  /// the height of the [hintText], even if the [hintText] is not visible.
+  ///
+  /// This is useful, If true, while inputting it will ignore the height of the [hintText]
```

Comment:
If you change the flag name to `mantainHintHeight`: Maybe: `The [InputDecorator] widget ignores [hintText] during layout when it's not visible, if this flag is set to false.`

Reply:
@LongCatIsLooong  Thx for your review, so mantainHintHeight default to true?  i update it ~

Reply:
Yeah. Also could you keep the first sentence?

Reply:
@LongCatIsLooong ok , I update

Reply:
Sorry I meant this:

```dart
  /// Whether the input field's height should always be greater than or equal to
  /// the height of the [hintText], even if the [hintText] is not visible.
  ///
  /// The [InputDecorator] widget ignores [hintText] during layout when
  /// it's not visible, if this flag is set to false.
  ///
  /// Defaults to true.
```